src/test/functional/functional-tests.js

```diff
@@ -126,21 +130,21 @@ describe('functional tests', function() {
 
   describe('makeBucket with period and region', () => {
     if (playConfig.endPoint === 's3.amazonaws.com') {
-      it('should create bucket in eu-central-1 with period', done => client.makeBucket(`${bucketName}.sec.period`,
+      step('makeBucket(bucketName, region, cb)_region:eu-central-1_', done => client.makeBucket(`${bucketName}.sec.period`,
                                                                                        'eu-central-1', done))
-      it('should delete bucket', done => client.removeBucket(`${bucketName}.sec.period`, done))
+      step('removeBucket(bucketName, cb)__', done => client.removeBucket(`${bucketName}.sec.period`, done))
     }
   })
 
   describe('listBuckets', () => {
-    it('should list bucket', done => {
+    step('listBuckets(cb)__', done => {
```

Comment:
This test case and the subsequent one doesn't depend on any previous case, is there a reason why you're changing it from `it` to `step`? (most of the others make sense)

Reply:
We want to exit the test right away if we see a failure and that is the reason, why we have moved away from `it` to `step` .